test/test_mkldnn.py

```diff
@@ -75,18 +76,38 @@ def test_detach(self):
         root = torch.randn(4, 5, dtype=torch.float32).to_mkldnn().requires_grad_()
 
         detach = root.detach()
-        self.assertEquals((4, 5), detach.size())
+        self.assertEqual((4, 5), detach.size())
```

Comment:
what's going on here?  Are both Equal and Equals usable?

Reply:
Yes. I change it because I got a warning saying Equals is being deprecated and here should change to Equal